src/components/contacts/NewContact.tsx

```diff
@@ -0,0 +1,273 @@
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useUserLoggedContext } from "../../context/UserLoggedContext";
+
+export interface FormData {
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+  companyId?: number | null;
+  notes: string;
+}
+
+const NewContact = () => {
+  const navigate = useNavigate();
+  
+  const { token } = useUserLoggedContext();
+
+  const [formData, setFormData] = useState<FormData>({
+    firstName: "",
+    lastName: "",
+    email: "",
+    phoneNumber: "",
+    companyId: null,
+    notes: "",
+  });
+
+  const [feedback, setFeedback] = useState<string | null>(null);
+  const [companies, setCompanies] = useState<{ id: number; name: string }[]>(
+    []
+  );
+
+  useEffect(() => {
```

Comment:
Pull this out the the trackerApiCall.tsx

Reply:
Great call out Jim, we have a ticket up, #62 as a refactor.